documentation/c03-input.sil

```diff
@@ -25,8 +25,8 @@ To manually change the paper size, an optional argument may be added to the docu
 \line
 \end{verbatim}
 
-SILE knows about the ISO standard A, B and C series paper sizes using names a4, b5, etc.
-Also available are presets for some traditional sizes:
+SILE knows about the ISO standard A, B and C series paper sizes using names like a4 and b5.
+Here is a list of presets for traditional sizes:
```

Comment:
```suggestion
SILE knows about the ISO standard A, B and C series paper sizes using names like a4 and b5 as well as many other traditional sizes.
Here is a full list of papersize preset names:
```

Reply:
Yes, that's much better than what I suggested 👍🏻